photoeditor/src/main/java/ja/burhanrashid52/photoeditor/OnPhotoEditorListener.java

```diff
@@ -58,4 +60,6 @@
      * @param viewType enum which define type of view is added
      */
     void onStopViewChangeListener(ViewType viewType);
+
+    void onTouchView(MotionEvent event);
```

Comment:
We can rename it to `onTouchSourceImage()`?

Reply:
Done, and I added some missing Javadoc on this method.